BUILD.bazel

```diff
@@ -1,13 +1,11 @@
-load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier")
-
 alias(
     name = "cargo_raze",
     actual = "//impl:cargo_raze",
     visibility = ["//visibility:public"],
 )
 
 alias(
-    name = "cargo_raze_bin",
+    name = "raze",
```

Comment:
why

Reply:
It shortens the name. I found `@cargo_raze//:cargo_raze_bin` to be a bit unnatural coming from `cargo raze` (the cargo equivalent. So I opted to rename the `bin` alias to `raze` to make it feel a bit more intuitive.

Reply:
I don't feel strongly about any of this, but if short is the criteria, `@cargo_raze//:bin` seems short and slightly less redundant.

Reply:
Can I add both? 😅 🙏

Reply:
🤷

Reply:
Done 😄